R/load_latest_forecasts_repo.R

```diff
@@ -127,6 +151,10 @@ load_forecast_files_repo <- function(file_paths,
                                      types = NULL,
                                      targets = NULL) {
   # validate file_paths exist
+  if (is.null(file_paths)){
```

Comment:
should we also check for `missing(file_paths)` here and/or provide a default of `NULL`?